controller/search_blackbox_test.go

```diff
@@ -1179,6 +1179,127 @@ func (s *searchControllerTestSuite) TestIncludedParents() {
 	})
 }
 
+func (s *searchControllerTestSuite) TestIncludedChildren() {
+	resetFn := s.DisableGormCallbacks()
+	defer resetFn()
+
+	// Suppose we have this topology:
+	//
+	//   A
+	//   |_B
+	//     |_C
+	//     | |_D
+	//     |
+	//     |_E
+	//
+	// We need to make sure that E is included when C is a match AND the
+	// tree-view option is "on". If C is a match and trew-view option is "off",
+	// then don't include E.
+	fxt := tf.NewTestFixture(s.T(), s.DB,
+		tf.WorkItems(5, tf.SetWorkItemTitles("A", "B", "C", "D", "E")),
+		tf.WorkItemLinksCustom(4,
+			tf.BuildLinks(append(tf.LinkChain("A", "B", "C", "D"), tf.L("B", "E"))...),
+			func(fxt *tf.TestFixture, idx int) error {
+				fxt.WorkItemLinks[idx].LinkTypeID = link.SystemWorkItemLinkTypeParentChildID
+				return nil
+			},
+		),
+	)
+
+	A := fxt.WorkItemByTitle("A").ID
+	B := fxt.WorkItemByTitle("B").ID
+	C := fxt.WorkItemByTitle("C").ID
+	E := fxt.WorkItemByTitle("E").ID
+	spaceIDStr := fxt.Spaces[0].ID.String()
+
+	testFolder := "in_topology_A-B-C-D_and_B-E_search_for_B,C"
```

Comment:
The `,` character is not MS Windows friendly for a folder name. How about replacing that with underscore?

Reply:
Thank you @baijum . I've addressed that in ab55be5